com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfInertial.cs

```diff
@@ -14,91 +14,128 @@
 
 using System;
 using UnityEngine;
+using UnityEngine.Assertions;
 
 namespace Unity.Robotics.UrdfImporter
 {
     [RequireComponent(typeof(ArticulationBody))]
     public class UrdfInertial : MonoBehaviour
     {
-        public bool displayInertiaGizmo;
-
+        const int k_RoundDigits = 10;
+        const float k_MinInertia = 1e-6f;
+        const float k_MinMass = 0.1f;
```

Comment:
What's the minimum mass based on here? We already have clamping in native `m_Mass = clamp<float>(mass, 0.0000001f, 1000000000);` Unless there's some specific minimum mass related to robotics?

Reply:
These are just renames/re-ordering of old lines 31-33.  const values to be prefixed with `k_` and declared above non-const member variables. I'm not sure what informs the minimums but if I'd have to guess they're just arbitrary above-zero values.